modules/op-improving-performance.adoc

```diff
@@ -0,0 +1,19 @@
+// This module is included in the following assemblies:
+// * resource/managing-pipelines-performance.adoc
+
+:_content-type: PROCEDURE
+[id="improving-performance_{context}"]
+= Improving {pipelines-shortname} performance
+
+If you experience slowness or recurrent failures of pipeline runs, you can take any of the following steps to improve the performance of {pipelines-shortname}.
+
+* Increase the number of nodes in the {OCP} cluster on which {pipelines-shortname} runs.
++
+[NOTE]
+====
+Increasing the CPU and RAM of individual nodes does not affect performance significantly.
+====
+
+* To enable the high-availability mode, in the `TektonConfig` custom resource (CR), set the `pipeline.performance.disable-ha` spec to `false`.
```

Comment:
with HA enabled, we saw the following 2 metrics improve:
- time taken to complete pipelineruns was reduced by a significant amount
- taskrun creation to pod creation latency reduced by a significant amount